tests/conftest.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+from rdkit import Chem
+
+from chemprop.data.datapoints import MoleculeDatapoint
+
+_DATA_DIR = Path(__file__).parent / "data"
+_DF = pd.read_csv(_DATA_DIR / "smis.csv")
+_DF["mol"] = _DF["smiles"].map(Chem.MolFromSmiles)
+_DF["smi"] = _DF["mol"].map(Chem.MolToSmiles)
+
+
+@pytest.fixture
+def data_dir():
+    return _DATA_DIR
+
+
+@pytest.fixture
+def smis():
+    return _DF.smi
+
+
+@pytest.fixture
+def mols():
+    return _DF.mol
+
+
+@pytest.fixture
+def targets(smis):
+    return np.random.rand(len(smis), 1)
+
+
+@pytest.fixture
+def mol_data(mols, targets):
+    return [MoleculeDatapoint(mol, y) for mol, y in zip(mols, targets)]
+
+
+@pytest.fixture
+def rxn_data(rxns, targets):
+    return [MoleculeDatapoint(mol, y) for mol, y in zip(mols, targets)]
+
+
+@pytest.fixture(params=_DF.smi.sample(5))
+def smi(request):
+    return request.param
+
+
+@pytest.fixture(params=_DF.mol.sample(5))
+def mol(request):
+    return request.param
+
+
+@pytest.fixture
+def mol_regression_data(data_dir):
+    df = pd.read_csv(data_dir / "regression/mol.csv")
+    smis = df["smiles"].to_list()
+    Y = df["lipo"].to_numpy().reshape(-1, 1)
+
+    return smis, Y
+
+
+@pytest.fixture
+def rxn_regression_data(data_dir):
+    df = pd.read_csv(data_dir / "regression/rxn.csv")
+    smis = df["smiles"].to_list()
+    Y = df["ea"].to_numpy().reshape(-1, 1)
+
+    return smis, Y
+
+
+@pytest.fixture
+def mol_mol_regression_data(data_dir):
```

Comment:
I don't see `mol_mol_regression_data` and `rxn_mol_regression_data` used. Do you expect that we will have an integration test for them in the future?

Reply:
yes, when someones writes integration tests for the `MulticomponentMPNN` class.